src/Features/CSharp/Portable/BraceCompletion/BracketBraceCompletionService.cs

```diff
@@ -30,5 +37,41 @@ public override Task<bool> AllowOverTypeAsync(BraceCompletionContext context, Ca
         protected override bool IsValidOpeningBraceToken(SyntaxToken token) => token.IsKind(SyntaxKind.OpenBracketToken);
 
         protected override bool IsValidClosingBraceToken(SyntaxToken token) => token.IsKind(SyntaxKind.CloseBracketToken);
+
+        protected override ImmutableArray<AbstractFormattingRule> GetBraceFormattingIndentationRulesAfterReturn(DocumentOptionSet documentOptions)
+        {
+            return ImmutableArray.Create(BraceCompletionFormattingRule.Instance);
+        }
+
+        private sealed class BraceCompletionFormattingRule : BaseFormattingRule
```

Comment:
>BraceCompletionFormattingRule [](http://example.com/codeflow?start=29&length=29)

Typo? Should this be called `BracketCompletionFormattingRule` or maybe just `FormattingRule`?